src/index.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import "./index.css";
+import Home from "./pages/Home";
+import Video from "./pages/Video";
+import { RouterProvider, createBrowserRouter } from "react-router-dom";
+
+const router = createBrowserRouter([
+  {
+    path: "/",
+    element: <Home />,
+  },
+  {
+    path: "/video/:id",
+    element: <Video />,
+  },
+]);
+
+const root = ReactDOM.createRoot(document.getElementById("root"));
```

Comment:
Add a comment above this line explaining that this is boilderplate and thus they should not worry about it